plugins/mcreator-core/procedures/entity_get_shootpower.json

```diff
@@ -0,0 +1,27 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "projectile_entity",
+      "check": "Entity"
+    }
+  ],
+  "output": "Number",
+  "inputsInline": true,
+  "colour": "%{BKY_MATH_HUE}",
+  "mcreator": {
+    "toolbox_id": "entitymanagement",
+    "toolbox_init": [
+      "<value name=\"projectile_entity\"><block type=\"immediate_source_entity_from_deps\"></block></value>"
+    ],
+    "inputs": [
+      "projectile_entity"
+    ],
+    "dependencies": [
+      {
+        "name": "imediatesourceentity",
+        "type": "imediatesourceentity"
```

Comment:
Test fails due to this mistake.

```
/tmp/mcreator_test_workspace2950262608553387228/src/main/java/net/mcreator/test_mod/procedures/TestBlockentity_get_shootpowerProcedure.java:23: error: cannot find symbol
    		return ((imediatesourceentity.getEntity() instanceof IProjectile) ? ((imediatesourceentity).getMotion().distanceTo(Vec3d.ZERO)) : 0);
```

change type to "Entity". This is still Entity variable, just with different name.

Reply:
Thank you. I was looking for the mistake but haven't found it. Thanks, it's been fixed now.